community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/StatisticsBackedCardinalityModel.scala

```diff
@@ -79,6 +79,10 @@ class StatisticsBackedCardinalityModel(queryGraphCardinalityModel: QueryGraphCar
     case _: UnwindProjection =>
       in * Multiplier(10)
 
+    // CallProcedure
+    case _: ProcedureCallProjection =>
+      in * Multiplier(10)
```

Comment:
I disagree on this estimation, it cannot be only based on the input. If input cardinality is zero this might cause bad planning of the remaining query. How about the `max(in * 10, 10000)`?